src/rollup.ts

```diff
@@ -59,6 +59,10 @@ export async function bundleStyles({
   });
   let text = result.outputFiles[0].text;
   if (path === getClientPath("stdlib/inputs.css")) text = rewriteInputsNamespace(text);
+  // dirty patch for tailwind: remove margin:0 and styles resets for headers
+  // etc. It should probably be a tailwind plugin instead.
+  if (path === getClientPath("tailwind.css"))
+    text = text.replaceAll(/}[^{]*h1,\n*h2,\n*h3,\n*h4,\n*h5,\n*h6[^}]+}\s*/g, "}");
```

Comment:
You want the [typography plugin](https://github.com/tailwindlabs/tailwindcss-typography) and `prose` class.

Reply:
Maybe, but there are other resets that wreck our page, and we might want to find a way to remove the resets independently of adding /prose/ (this [works](https://github.com/observablehq/framework/pull/1762#issuecomment-2437441570) already, anyway). I was thinking maybe a custom plugin we'd inline.

Reply:
Yes that's a good possibility. For now I'm merely taking stock of the things that change.

The other big change I've noticed are the `grid-cols-*` classes, which I have to block for now because they are quite different. Ours is responsive to width, tailwind's responsiveness is based on a breakpoint modifier (_e.g._ `lg:grid-cols-3`).

`blocklist: ["grid", "grid-cols-2", "grid-cols-3", "grid-cols-4"],`